Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -500,10 +500,9 @@ def _invoke_states_cb(self, callback):
             ds = d.network_get_state(None)
             dslist.append(ds)
 
-        def on_cb_done(new_get_peer_list):
-            self.get_peer_list = new_get_peer_list
-
-        return deferToThread(callback, dslist).addCallback(on_cb_done)
+        new_get_peer_list = callback(dslist)
```

Comment:
Putting the session callback on the reactor thread makes sense, but are we sure we also want to execute the 38000 SQL statements from the bootstrap file on the reactor thread? It takes 3-5 seconds on my workstation.